### PR TITLE
Recipes for RockPi SBCs

### PR DESCRIPTION
**Pull request by Darmur**
- updated recipe for RockPi-S, previous one was not working due to missing platform repo
- new recipe for RockPi-E

### CODE REVIEW

**Comment by ashthespy** [comment]
I stopped adding/updating/maintaining these boards as after the requirement of myVolumio accounts.
I believe @gkkpch also dropped support for Odriod and Vim boards a while back as well.

No myVolumio automatically means no plugins, manifestUI, OTA updater, etc, etc.

So community portings are quite dead IMO, and can be removed from this repo?

**Comment by gkkpch** [comment]
Community portings only make sense when they can have myVolumio integrated (means compiling on the Volumio server), otherwise they are not much more than a webradio and airplay build for personal use. 

A porting can work well though, provided there are enough users appreciating the work and being able to make the distinction between getting a porting for free and paying for a myVolumio account. 
Some users couldn't and I had a disagreement with them and that is why I stopped.  
Only the Odroid C4 and N2 are kept alive because they work so well with Volumio.
The nanopi neo3 and neo2 are kept up-to-date because of OEM dependencies. 

The rest could be flushed ;)

**Comment by Darmur** [comment]
please don't flush them! I am learning the process, I'd like to take-over the maintenance of some community porting in the future, I have working images of those boards (local build without myvolumio). I'm also adding support for some other boards I have at home, even if they are old they work pretty well with Volumio.

**Comment by ashthespy** [comment]
Adding devices is quite easy with the current build system.But with most features dependent on myVolumio, that community portings cant use, I also don't see the point of maintaing them? A small minority of people run Volumio on these boards. 


MyVolumio needs a dedicated volumio team member to build and maintain images with the propriety bits. This isn't a trivial amount of work, and I think the devs have quite a lot on their plate already...

Simply put, what is the end goal?

**Comment by Darmur** [comment]
I don't think triggering a build from the main server from time to time (every two months?) is a big amount of work, I think I can take care of it.

**Comment by ashthespy** [comment]
Then please also move the platform files under the volumio account, so a repeat of what happened with mine doesn't happen! :-)

**Comment by Darmur** [comment]
already asked to fork my repo, waiting for the big boss to be back in the office :)

**Comment by volumio** [comment]
Back and merged!
